dbt/adapters/bigquery/connections.py

```diff
@@ -304,6 +304,16 @@ def get_google_credentials(cls, profile_credentials) -> GoogleCredentials:
                 scopes=profile_credentials.scopes,
             )
 
+        elif method == BigQueryConnectionMethod.AWS_FILE:
+            aws_creds = GoogleAuthAWS.Credentials
+            details = profile_credentials.keyfile
+            return aws_creds.from_file(details, scopes=profile_credentials.scopes)
+
+        elif method == BigQueryConnectionMethod.AWS_FILE:
+            aws_creds = GoogleAuthAWS.Credentials
+            details = profile_credentials.keyfile
+            return aws_creds.from_file(details, scopes=profile_credentials.scopes)
```

Comment:
Am I missing something or is this `elif` block repeated as is?

Reply:
It does look duplicated. I'll resolve that.

Reply:
removed in the latest push.